<?xml version="1.0"?>
<phpunit xmlns="http://schema.phpunit.de/coverage/1.0">
  <file name="AlmaException.php">
    <totals>
      <lines total="33" comments="22" code="11" executable="2" executed="1" percent="50.00%"/>
      <methods count="0" tested="0" percent=""/>
      <functions count="0" tested="0" percent=""/>
      <classes count="0" tested="0" percent=""/>
      <traits count="0" tested="0" percent=""/>
    </totals>
    <class name="AlmaException" start="31" executable="0" executed="0" crap="0">
      <package full="" name="" sub="" category=""/>
      <namespace name="Alma\PrestaShop\Exceptions"/>
    </class>
    <coverage>
      <line nr="27">
        <covered by="Alma\PrestaShop\Tests\Unit\Helper\Admin\InsuranceHelperTest::testDontSaveConfigInsuranceWithWrongKeysAndThrowException"/>
        <covered by="Alma\PrestaShop\Tests\Unit\Helper\MessageOrderHelperTest::testGetMessageForRefundWithWrongData"/>
      </line>
    </coverage>
  </file>
</phpunit>
